test/integration/dialects/sqlite/dao.test.js

```diff
@@ -72,6 +75,223 @@ if (dialect === 'sqlite') {
       });
     });
 
+    describe('json', function() {
+      it('should tell me that a column is json', function() {
+        return this.sequelize.queryInterface.describeTable('Users')
+          .then(function(table) {
```

Comment:
OK, since I cannot change all the es5-styled files, I will convert at least this file to es6.